src/main/scala/com/sumologic/sumobot/core/Bootstrap.scala

```diff
@@ -58,6 +71,31 @@ object Bootstrap {
     sys.addShutdownHook(shutdownActorSystem())
   }
 
+  private def bootstrapHttp(brainProps: Props, pluginCollections: Seq[PluginCollection]): Unit = {
+    val httpConfig = system.settings.config.getConfig("http")
+    val httpHost = httpConfig.getString("host")
+    val httpPort = httpConfig.getInt("port")
+
+    val brain = system.actorOf(brainProps, "brain")
+    val httpServer = new SumoBotHttpServer(httpHost, httpPort)
+
+    receptionist = Some(system.actorOf(Props(classOf[HttpReceptionist], brain), "receptionist"))
+
+    pluginCollections.par.foreach(_.setup)
+
+    sys.addShutdownHook(shutdownActorSystem())
+  }
+
+  private def selectedFrontend(): String = {
+    val isSlackSelected = system.settings.config.hasPath("slack.api.token")
+    val isHttpSelected = system.settings.config.hasPath("http")
+
+    if (isHttpSelected && isSlackSelected) throw new RuntimeException("Only one frontend can be selected")
```

Comment:
more suited would be `IllegalArgumentException`

Reply:
Fixed in: https://github.com/SumoLogic/sumobot/pull/41/commits/8cbbea3e70ad90723787ec0bd88efe4b8d10c6ce